pints/_mcmc/_mala.py

```diff
@@ -166,9 +165,6 @@ def ask(self):
         if not self._running:
             self._initialise()
 
-        if self._ready_for_tell:
-            raise RuntimeError('Ask() called when expecting call to tell().')
-
         # Propose new point
```

Comment:
Not sure I follow why we'd want to remove this check? Probably missing the point, but what would be a use case for repeatedly calling ask?